js/bit4you.js

```diff
@@ -0,0 +1,624 @@
+                const canceled = this.safeInteger (transaction, 'canceled');
+                let status = '';
+                if (!pending && !canceled) {
+                    status = 'ok';
+                } else {
+                    status = 'canceled';
+                }
+                const amount = this.safeNumber (transaction, 'quantity');
+                const currency = this.safeString (transaction, 'iso');
+                const updated = this.safeInteger (transaction, 'update_time');
+                const comment = null;
+                const fee = {
+                    'currency': this.safeString (transaction, 'fee_iso'),
+                    'cost': this.safeNumber (transaction, 'fee'),
+                    'rate': undefined,
+                };
+                result.push ({
+                    'info': transaction,
+                    'id': this.safeString (transaction, 'id'),
+                    'txid': this.safeString (transaction, 'txid'),
+                    'timestamp': timestamp,
+                    'datetime': datetime,
+                    'address': this.safeString (transaction, 'address'),
+                    'tag': this.safeString (transaction, 'tag'),
+                    'type': type,
+                    'amount': amount,
+                    'currency': currency,
+                    'status': status,
+                    'updated': updated,
+                    'comment': comment,
+                    'fee': fee,
+                });
+            }
+            return result;
+        } catch (error) {
+            return error;
+        }
+    }
+
+    async withdraw (code, amount, address, tag = undefined, params = {}) {
+        const request = {};
+        request.iso = code;
+        request.quantity = amount;
+        if (tag) {
+            request.addres = address + ':' + tag;
+        } else {
+            request.address = address;
+        }
+        request.chain = params.network + '-mainnet' || '';
+        try {
+            const response = await this.privatePostWalletSend (this.extend (request));
+            return response || [];
+        } catch (error) {
+            return error;
+        }
+    }
+
+    async fetchStatus (params = {}) {
+        return {
+            'status': 'ok', // 'ok', 'shutdown', 'error', 'maintenance'
+            'updated': this.milliseconds (), // integer, last updated timestamp in milliseconds if updated via the API
+            'eta': undefined, // when the maintenance or outage is expected to end
+            'url': undefined, // a link to a GitHub issue or to an exchange post on the subject
+        };
+    }
+
+    async fetchTicker (symbol = undefined, params = {}) {
+        await this.loadMarkets ();
+        const cached = this.market (symbol);
+        const market = cached['symbol'];
+        try {
+            const response = await this.fetchTickers ();
+            return response[market] || [];
```

Comment:
You need to parse the response into our unified structure, check how it's done here:

https://github.com/ccxt/ccxt/blob/master/js/hollaex.js#L430
https://github.com/ccxt/ccxt/blob/master/js/hollaex.js#L454